client/src/containers/home-page/logic/saga.ts

```diff
@@ -24,6 +25,16 @@ export function* fetchMessagesWatcher() {
 	yield takeEvery(actionTypes.GET_MESSAGES, fetchMessagesWorker);
 }
 
+export function* fetchCommunityWorker(action: ReturnType<typeof actions.getCommunity>): any {
+	const { id } = action;
+	const { community } = yield call(() => fetchCommunity(id));
```

Comment:
why not simply write 
`const { community } = yield call(fetchCommunity(id));`
?

Reply:
because I have to pass a callback function
if I write simply fetchCommunity(id) it'll immediately invoke this function which returns an object